builtins/amp-ad.js

```diff
@@ -217,6 +223,12 @@ export function installAd(win) {
       assert(!this.isInFixedContainer_,
           '<amp-ad> is not allowed to be placed in elements with ' +
           'position:fixed: %s', this.element);
+      if (this.isResizable_) {
+        this.element.setAttribute('scrolling', 'no');
+        assert(this.getOverflowElement(),
```

Comment:
This is ok for now. But I think in reality this will never be useful for ads. Most likely request to anticipate: instead of showing overlfow element Ads developers will ask a signal back when we couldn't expand. E.g. a postMessage with `embed-size-denied`. Then they can provide their own overflow element within and repeat request on tap. In that case AMP will definitely allow size change due to user action. 

This would be useful for both: iframes and ads. I'm ok if we file a bug to do this for now. But I'd leave it to @cramforce to provide a final word on this.